app/factory/Layer.js

```diff
@@ -106,9 +106,77 @@ Ext.define('CpsiMapview.factory.Layer', {
         });
     },
 
+    /**
+     * Creates an OGC WFS layer
+     *
+     * @param  {Object} layerConf The configuration object for this layer
+     * @return {ol.layer.Vector}  WFS layer
+     */
     createWfs: function(layerConf) {
 
-        Ext.log.info('Not implemented yet', layerConf);
+        var url = layerConf.url;
+        if (url.indexOf() == -1) {
+            url+= '?';
```

Comment:
nit-pick, space missing
```suggestion
            url += '?';
```